core/src/main/java/org/elasticsearch/common/logging/DeprecationLogger.java

```diff
@@ -31,6 +34,12 @@
  */
 public class DeprecationLogger {
 
+    private final Logger logger;
+
+    public Logger getLogger() {
```

Comment:
Thanks for noticing this @nik9000. This is leftover from a debugging session. :smile: I pushed 0fdc5ca5876673f08c3c69ef99f9e18d22f3a9b4.